typescript/infra/src/core/govern.ts

```diff
@@ -109,20 +168,11 @@ export class AbacusCoreGovernor<Chain extends ChainName> {
     }
   }
 
-  // This function is an exception in that it assumes the MultiProvider
-  // is configured with the privileged signers. All other functions assume
-  // governance is done via multisig.
   async handleOwnerViolation(violation: OwnerViolation) {
-    const chainConnection = this.checker.multiProvider.getChainConnection(
-      violation.chain as Chain,
-    );
-    console.log(
-      `${violation.chain}: transferring ownership of ${violation.data.contract.address} from ${violation.actual} to ${violation.expected}`,
-    );
-    const response = await violation.data.contract.transferOwnership(
-      violation.expected,
-      chainConnection.overrides,
-    );
-    await response.wait(chainConnection.confirmations);
+    const call =
+      await violation.data.contract.populateTransaction.transferOwnership(
+        violation.expected,
+      );
+    this.pushCall(violation.chain as Chain, call as types.CallData);
```

Comment:
- checker produces violations
- infra governor produces calls

boundaries here are still a little unclear
seems like this stuff shouldnt be buried in infra scripts